File: example/javalib/testing/5-test-grouping/foo/test/src/foo/HelloTests.java
package foo;

import static org.junit.Assert.assertTrue;

import org.junit.Test;

public class HelloTests {

  @Test
  public void hello() throws Exception {
    System.out.println("Testing Hello");
    String result = new Foo().hello();
    assertTrue(result.startsWith("Hello"));
    Thread.sleep(1000);
    System.out.println("Testing Hello Completed");
  }
}

File: example/kotlinlib/testing/4-test-grouping/foo/test/src/foo/HelloTests.kt
package foo

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.string.shouldStartWith

class HelloTests :
    FunSpec({
        test("hello") {
            println("Testing Hello")
            val result = Foo().hello()
            result shouldStartWith "Hello"
            java.lang.Thread.sleep(1000)
            println("Testing Hello Completed")
        }
    })
